readme.md

```diff
@@ -126,12 +126,22 @@ You may also pass a sub-application, which _must_ be accompanied by a `base` pat
 Please see [`Middleware`](#middleware) and [Express' middleware examples](http://expressjs.com/en/4x/api.html#middleware-callback-function-examples) for more info.
 
 
-### parse(req)
+### parse(req, toDecode=true)
 
 Returns: `Object` or `undefined`
 
 As of `v0.5.0`, this is an alias of the [`@polka/url`](/packages/url) module. For nearly all cases, you'll notice no changes.
 
+One important difference is that URL decoding is enabled by default in polka but it is disabled by default in the `@polka/url` module. You can disable the URL decoding if required,
+
+```js
+const app = polka();
+const parse = require('@polka/url');
```

Comment:
This didn't work when I tried it. It should be:
```suggestion
const { parse } = require('@polka/url');
```

Reply:
No – this is for `README` on `master` which is the `polka@0.5.2` release, which is still using `module.exports` instead of a named export.

Reply:
Ah, nevermind. I did it with `@next`

Though I only saw Polka decode the URL by default with `@next`, so I'm not sure this section would be relevant otherwise